src/Amazon.Lambda.Tools/LambdaPackager.cs

```diff
@@ -92,7 +99,7 @@ private static bool IsAmazonLinux2(IToolLogger logger)
             else if (string.IsNullOrWhiteSpace(containerImageForBuild))
             {
                 // Use a default container image if Use Container is set to true, or if we need to build NativeAOT on non-AL2
```

Comment:
Could update this comment to say non-AmazonLinux

Reply:
I don't have permissions to approve the PR, but this comment is a nit so otherwise, looks good to me.